bigtable-client-core-parent/bigtable-client-core/src/main/java/com/google/cloud/bigtable/grpc/async/AbstractRetryingOperation.java

```diff
@@ -434,6 +434,11 @@ protected RequestT getRetryRequest() {
   public ListenableFuture<ResultT> getAsyncResult() {
     Preconditions.checkState(operationTimerContext == null);
     operationTimerContext = rpc.getRpcMetrics().timeOperation();
+
+    // CreateFirstAttempt establishes the time when first call was made and the deadline is set to `timeOfFirstCall +
+    // timeout`. Hence, its important to create first attempt before any RPCs go out of client.
+    currentBackoff = exponentialRetryAlgorithm.createFirstAttempt();
```

Comment:
Since currentBackoff is never null now, can you remove the if null check L279?

Reply:
Done.